src/sidebar/store/modules/drafts.js

```diff
@@ -178,11 +180,10 @@ function getDraftIfNotEmpty(state, annotation) {
  *
  * @return {Object[]}
  */
-function unsavedAnnotations(state) {
-  return state.drafts
-    .filter(draft => !draft.annotation.id)
-    .map(draft => draft.annotation);
-}
+const unsavedAnnotations = createSelector(
+  state => state.drafts,
+  drafts => drafts.filter(d => !d.annotation.id).map(d => d.annotation)
+);
```

Comment:
This is just an optimization of the previous selector to make it more efficient.